DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -4259,8 +4257,6 @@
 				83ED3B8D1FA8E63700B47556 /* README.md */,
 				83ED3B8C1FA8E61D00B47556 /* ManualTestsScript.md */,
 				85A313962028E78A00327D00 /* release_notes.txt */,
-				983D88D52CED006000E99B46 /* UnitTests copy-Info.plist */,
-				98424AAA2CED4F430071C7DB /* UnitTests copy-Info.plist */,
```

Comment:
These appear to be from a bad merge unrelated to this PR; cleaning them up here.